route/build/src/write_restart_pio.f90

```diff
@@ -1235,6 +1347,41 @@ SUBROUTINE write_DW_state(ierr, message1)
 
   END SUBROUTINE write_DW_state
 
+  SUBROUTINE write_tracer_state(idxRoute, ierr, message1)
```

Comment:
Description.

Reply:
subroutine name is changed to `write_solute_state`. The same question as above.